docs/pages/api-docs/date-picker.json

```
@@ -86,7 +86,7 @@
     "showTodayButton": { "type": { "name": "bool" } },
     "showToolbar": { "type": { "name": "bool" } },
     "todayText": { "type": { "name": "node" }, "default": "\"TODAY\"" },
-    "ToolbarComponent": { "type": { "name": "func" }, "default": "DatePickerToolbar" },
+    "ToolbarComponent": { "type": { "name": "elementType" }, "default": "DatePickerToolbar" },
```

Should it be documented as?

```suggestion
    "ToolbarComponent": { "type": { "name": "componentType" }, "default": "DatePickerToolbar" },
```

I mean, should the docs communicate if native element are accepted or not?